config/opal_check_pmi.m4

```diff
@@ -11,23 +11,19 @@
 # Copyright (c) 2004-2005 The Regents of the University of California.
 #                         All rights reserved.
 # Copyright (c) 2009-2015 Cisco Systems, Inc.  All rights reserved.
-# Copyright (c) 2011-2016 Los Alamos National Security, LLC. All rights
+# Copyright (c) 2011-2014 Los Alamos National Security, LLC. All rights
```

Comment:
Are we going to a previous version? Sorry if I missed this discussion.

Reply:
My bad.. just saw the rest of the content...

Reply:
It is kinda funny... maybe a small mistake?